src/main/java/com/thebois/persistence/SaveSystem.java

```diff
@@ -0,0 +1,50 @@
+package com.thebois.persistence;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+/**
+ * Handles the saving of objects to the save-file.
+ */
+public class SaveSystem extends AbstractPersistence {
+
+    private final ObjectOutputStream objectOutputStream;
+
+    /**
+     * Creates an instance of an output stream for the save-file.
+     *
+     * @throws IOException If the file save-file is deleted during runtime.
+     */
+    public SaveSystem() throws IOException {
+
+        directoryExist();
```

Comment:
Here you can call `super();`

Reply:
Once again see the first comment about this